### PR TITLE
Created works_about_myself_sections

### PR DESCRIPTION
**Pull request by SebastianSolisFenger**
I created the mobile website section where I will list my portfolio projects.
Furthermore, I worked on:

- Understanding how to parse a Figma design to create a UI.
- Using Flexbox to place elements on the page.
- Using CSS Grid to place elements on the page.

### CODE REVIEW

**Comment by SebastianSolisFenger** [comment]
Hi! Hope you're doing well today. Here's my pull request.

On the other hand, would you be so kind of telling me why could the extension PRETTIER has stopped working on my Vscode? I really googled everything but still not working, though it used to work perfectly well. 
Could it be because of a file? I cannot make it work on Html. file and I don't know HOW TO INDENT in there.
Thanks.
Best, Seba.

File: style.css [hunk 1]
```diff
@@ -218,3 +218,340 @@ body {
...
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+}
```
Comment (RolandM99) [inline style.css]: - [x] Make sure that you follow correctly the design from Figma to implement yours. 
### Your Design:
image
### Figma design:
image

To point the Figma design, you can remove the border from those buttons by adding `border: none;` in the code above.

File: index.html [hunk 2]
```diff
@@ -73,6 +79,272 @@ <h1>I’m Sebastian. Glad to see you!</h1>
...
+          <li>
+            <div class="dropdownlink bottom-separator">
+              <h3 class="about-titles" id="sk-title">Skills</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"> </i>
+            </div>
+            <!-- <ul class="submenuItems">
+                <li><a href="#">Fantasy book 1</a></li>
+                <li><a href="#">Fantasy book 2</a></li>
+                <li><a href="#">Fantasy book 3</a></li>
+              </ul> -->
```
Comment (RolandM99) [inline index.html]: - [x] To make your code clear and clean, please kindly, remove the commented codes in your repo.

File: index.html [hunk 3]
```diff
@@ -73,6 +79,272 @@ <h1>I’m Sebastian. Glad to see you!</h1>
...
+            <li>
+              <a href="#"> <i class="fa fa-github"></i> </a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin"></i></a>
+            </li>
+          </ul>
+        </div>
+
+        <button class="see-proj-btn-itself resume-btn" type="button">
```
Comment (RolandM99) [inline index.html:265]: - [x] All buttons on this project should respect and follow the button styling: Active, Hover, and Pressed (see design below):
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @SebastianSolisFenger  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Comment by RolandM99** [comment]
Hello @SebastianSolisFenger, 
Kindly read this [documentation](https://stackoverflow.com/questions/52586965/why-does-prettier-not-format-code-in-vs-code#:~:text=You%20just%20need%20to%20press,json%20file.&text=This%20worked%20for%20me%2C%20thanks!), I think that it can be helpful for you about the issue with your PRETTIER extension.

**Comment by SebastianSolisFenger** [comment]
> Hello @SebastianSolisFenger, Kindly read this [documentation](https://stackoverflow.com/questions/52586965/why-does-prettier-not-format-code-in-vs-code#:~:text=You%20just%20need%20to%20press,json%20file.&text=This%20worked%20for%20me%2C%20thanks!), I think that it can be helpful for you about the issue with your PRETTIER extension.

Thank you, I had already check that page actually, but I don't know what's going on, yet.

File: index.html [hunk 4]
```diff
@@ -73,6 +70,253 @@ <h1>I’m Sebastian. Glad to see you!</h1>
...
+
+          <li>
+            <div class="dropdownlink bottom-separator">
+              <h3 class="about-titles" id="fr-title">Frameworks</h3>
+              <i class="fa fa-chevron-down arrows" aria-hidden="true"></i>
+            </div>
+          </li>
+
+          <li>
+            <div class="dropdownlink bottom-separator">
```
Comment (vic778) [inline index.html]: - [OPTIONAL] kindly make sure to use the horizontal flesh when the link is not clickable as we can see in the [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/about-myself-mobile.png) template

**Review by vic778 — APPROVED** [review]
### STATUS: APPROVED :+1:  
Hi @SebastianSolisFenger, 

Good job so far!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Descriptive commits and correct use of the Git Flow :fire:
- [x] :white_check_mark: page titles :white_check_mark: text headings :white_check_mark: color contrast :white_check_mark: resize :white_check_mark: basic structure of the page 

## Required Changes ♻️
N/A

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (SebastianSolisFenger) [reply index.html]: Ok, already done it!

**Pull request merged**